terms.php

```diff
@@ -1 +1,33 @@
    $player = User::loadFromId($system, $_SESSION['user_id']);
    $player->loadData();
    $layout = $system->fetchLayoutByName($player->layout);
}

$layout->renderStaticPageHeader('Terms of Service');
	
?>

<table class='table'><tr><th>Terms of Service</th></tr>
<tr><td>
<!--START_EDIT-->

Shinobi-chronicles.com is a fan site: We did not create Naruto nor any of the characters and content in Naruto. While inspired by 
Naruto, the content of this site is fan-made and not meant to infringe upon any copyrights, it is simply here to further the 
continuing popularity of Japanese animation. In no event will shinobi-chronicles.com, 
its host, and any other companies and/or sites linked to shinobi-chronicles.com be liable to any party for any direct, indirect, 
special or other consequential damages for any use of this website, or on any other hyperlinked website, including, without limitation,
 any lost profits, loss of programs or other data on your information handling system or otherwise, even if we are expressly advised 
 of the possibility of such damages.<br />

<p>
Shinobi-chronicles.com accepts no responsibility for the actions of its members i.e. Self harm, vandalism, suicide, homicide, 
genocide, drug abuse, changes in sexual orientation, or bestiality. Shinobi-chronicles.com will not be held responsible and does not 
encourage any of the above actions or any other form of anti social behaviour. The staff of shinobi-chronicles.com reserve the 
right to issue bans and/or account deletion for rule infractions. Rule infractions will be determined at the discretion of the 
moderating staff.</p>

<p>Loans or transactions of real or in-game currency are between players. Staff take no responsibility for the 
completion of them. If a player loans real or in-game currency to another player, staff will not be responsible for ensuring the 
currency is returned.</p>

<p>Ancient Kunai(Premium credits) that have already been spent on in-game purchases of any kind or traded to another player cannot be 
	refunded. Staff are not responsible for lost shards or time on Forbidden Seals lost due to user bans.</p>
<br />
The Naruto series is created by and copyright Masashi Kishimoto and TV Tokyo, all rights reserved.
</td></tr></table>
<?php
$layout->renderStaticPageFooter($player ?? null);
\ No newline at end of file
+<?php
+session_start();
+
+require "classes/System.php";
+$system = new System();
+$layout = $system->fetchLayoutByName(System::DEFAULT_LAYOUT);
+
+if(isset($_SESSION['user_id'])) {
+    require_once 'classes.php';
+    $player = User::loadFromId($system, $_SESSION['user_id']);
+    $player->loadData();
+    $layout = $system->fetchLayoutByName($player->layout);
+}
+
+if ($layout->key == "new_geisha") {
+    require($layout->headerModule);
+    echo $layout->heading;
+    require($layout->sidebarModule);
+    require($layout->topbarModule);
+    echo "<div id='content'>";
+} else {
+    $layout->renderStaticPageHeader('Terms of Service');
+}

```

Comment:
Hmm why not move this logic inside `$layout->renderStaticPageHeader()`?

Reply:
`$layout->renderStaticPageHeader()` fits here functionally but the the sidebar and topbar aren't conceptually part of a "Static Page Header". For a proper conversion I assume we would want to put those modules into their own functions, but the existing functions for them (e.g. `renderSideMenu()`) are not used in the same order between the old and new layouts.

In the meantime I could modify `$layout->renderStaticPageHeader()` to the following and add necessary System/Player variables as optional parameters:

```
public function renderStaticPageHeader(string $page_title, ?System $system = null, ?User $player = null): void {
        if ($this->key == "new_geisha") {
            require($this->headerModule);
            echo $this->heading;
            require($this->sidebarModule);
            require($this->topbarModule);
            return;
        }
        echo $this->heading;
        echo $this->top_menu;
        echo $this->header;
        echo str_replace("[HEADER_TITLE]", $page_title, $this->body_start);
    }
```